examples/adaptation.ipynb

```diff
@@ -0,0 +1,604 @@
+    "                TripPurposeEnum.BUSINESS,\n",
+    "                TripPurposeEnum.COMMUTE,\n",
+    "                TripPurposeEnum.FREIGHT,\n",
+    "                TripPurposeEnum.OTHER,\n",
+    "            ],\n",
+    "            resilience_curves_file=input_path.joinpath(\"resilience_curve.csv\"),\n",
+    "            traffic_intensities_file=input_path.joinpath(\"traffic_intensities.csv\"),\n",
+    "            values_of_time_file=input_path.joinpath(\"values_of_time.csv\"),\n",
+    "            save_gpkg=True,\n",
+    "            save_csv=True,\n",
+    "        )\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adaptation options:\n",
+    "\n",
+    "The adaptation options can now be defined. \n",
+    "\n",
+    "1. The `AnalysisSectionAdaptation` must be filled with general inputs applicable to all adaptation options, such as the discount rate, ...\n",
+    "2. A collection of adaptation options is to be specified:\n",
+    "    - Each adaptation option has an `id` which must match the input data structure in order to assign the input files correctly\n",
+    "    - The first adaptation option represents the initial situation (Business As Usual) and only requires a `name` and `id`.\n",
+    "    - The following adaptation options have extra required attributes to calculate the cost: `construction_cost`, `construction_interval`, 'maintenance_cost', 'maintenance_interval'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "# - adaptation\n",
+    "_adaptation_options = [\n",
+    "    AnalysisSectionAdaptationOption(\n",
+    "            id=\"AO0\",\n",
+    "            name=\"No adaptation\",\n",
+    "        ),\n",
+    "        AnalysisSectionAdaptationOption(\n",
+    "            id=\"AO1\",\n",
+    "            name=\"Cheap construction, expensive maintenance\",\n",
+    "            construction_cost=1000.0,\n",
+    "            construction_interval=10.0,\n",
+    "            maintenance_cost=200.0,\n",
+    "            maintenance_interval=3.0,\n",
+    "        ),\n",
+    "        AnalysisSectionAdaptationOption(\n",
+    "            id=\"AO2\",\n",
+    "            name=\"Expensive construction, cheap maintenance\",\n",
+    "            construction_cost=5000.0,\n",
+    "            construction_interval=100.0,\n",
+    "            maintenance_cost=50.0,\n",
+    "            maintenance_interval=3.0,\n",
+    "        ),\n",
+    "]\n",
+    "_adaptation_section = AnalysisSectionAdaptation(\n",
+    "        analysis=AnalysisEnum.ADAPTATION,\n",
+    "        losses_analysis=AnalysisLossesEnum.MULTI_LINK_LOSSES,\n",
+    "        adaptation_options=_adaptation_options,\n",
+    "        discount_rate=0.025,  # correcting inflation 0.025 = 2.5%\n",
+    "        initial_frequency=0.001,  # yearly frequency of occurrence of the event (hazard map)\n",
```

Comment:
Shouldn't we consider renaming it to probability?

Reply:
no because this frequency can be greater than 1